templates/solutions/secure-open-source/_form-secure-open-source.html

```diff
@@ -0,0 +1,83 @@
+<div class="p-modal" style="display: none;" id="secure-open-source-modal">
+  <div class="p-modal__dialog is-paper" role="dialog">
+    <div class="js-pagination js-pagination--1">
+      <div class="js-formfield">
+      <div class="row--50-50 p-strip is-shallow">
+        <div class="col">
+          <h2 class="p-heading--1">Get in touch with the Data & AI team</h2>
+          <p>Fill in this form and a member of our team will be in touch shortly.</p>
+        </div>
+        <div class="col">
+          <button class="p-modal__close p-modal-close-button " aria-controls="secure-open-source-modal" aria-label="Close active modal" style="margin-top: 0;">Close</button>
+          <form action="https://ubuntu.com/marketo/submit" method="post" class="modal-form">
+            <div class="p-section--shallow">
+              <label for="firstName" class="is-required">First name:</label>
+              <input required id="firstName" name="firstName" maxlength="255" type="text"/>
+            </div>
+            <div class="p-section--shallow">
+              <label for="lastName" class="is-required">Last name:</label>
+              <input required id="lastName" name="lastName" maxlength="255" type="text"/>
+            </div>
+            <div class="p-section--shallow">
+              <label for="email" class="is-required">Email:</label>
+              <input required id="email" name="email" maxlength="255" type="email" pattern="^[^ ]+@[^ ]+\.[a-z]{2,26}$" />
+            </div>
```

Comment:
The country field should be in between these instead of at the bottom of the form according to the doc